library/worker_registration.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2023 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package library
+
+// WorkerRegistration is the library representation of a WorkerRegistration.
+//
+// swagger:model WorkerRegistration
+type WorkerRegistration struct {
+	RegistrationToken *string `json:"registration_token,omitempty"`
+	QueuePublicKey    *string `json:"queue_public_key,omitempty"`
+	QueueAddress      *string `json:"queue_address,omitempty"`
+}
+
+// GetRegistrationToken returns the RegistrationToken field.
+//
+// When the provided Worker type is nil, or the field within
```

Comment:
```suggestion
// When the provided WorkerRegistration type is nil, or the field within
```
Probably from copy pasta from the Worker type. Should be changed in a number of places here. I'm only marking this instance.